scripts/RNA_analysis/analysis_report.py

```diff
@@ -665,6 +665,8 @@ def main(project_id,sample_names,single_end,config_file,Map_Stat,Read_Dist,FPKM,
     help = "Samplenames should be given as a coma delimited string. Default will be the samples given by the tophat_out - directories")
     parser.add_option( "-b", "--complexity", dest="complexity",action="store_true", default=False,
     help = "to include library complexity plot")
+    parser.add_option('-G', '--genome', action="store", dest="genome", default=None,
+    help="Reference genome name (eg. 'GRCh37') Default: fetch from status DB")
```

Comment:
@jun-wan can you list here all the supported genomes and the name to specify here to use them? This will simplify the use of this pipeline and make it more user friendly